tools/c7n_tencentcloud/pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.poetry]
+name = "c7n_tencentcloud"
+version = "0.1.0"
+description = "Cloud Custodian - Tencent Cloud Provider"
+authors = ["Tencent Cloud"]
+license = "Apache-2.0"
+
+[tool.poetry.dependencies]
+python = "^3.9"
+tencentcloud-sdk-python = "^3.0.715"
```

Comment:
what's the difference between this package and `tencentcloud-sdk-python-intl-en`

Reply:
Officially, tencentcloud-sdk-python-intl-en is a version with English annotations and this one's is in Chinese. But we have more experience on Chinese version and few with English version. Do we need to switch this package into English version?

Reply:
I would just want to confirm that the English and Chinese version are released in parallel and are one to one on object model. If that's the case I'm fine with proceeding with the Chinese version for development ease by the team, with the idea that a switch out at a later date is seamless.

Reply:
We will do a switch test to decide if it is necessary to switch this provider from the begging,  after our next pr with unit test coverage which will make this test easier.